api/envoy/extensions/filters/http/composite/v3/composite.proto

```diff
@@ -37,3 +37,9 @@ message Composite {
 message ExecuteFilterAction {
   config.core.v3.TypedExtensionConfig typed_config = 1;
 }
+
+
+message ExecuteFilterMultiAction {
```

Comment:
I feel we should have some better documentation here explaining what this is, why you would use it, how etc.